lib/giraph/remote/mutation.rb

```diff
@@ -0,0 +1,12 @@
+module Giraph
+  module Remote
+    # Field resolver to plug a remote GraphQL mutation root into a local type
+    class Mutation < Query
```

Comment:
The `Connector` seems to be the only thing that cares about the `@mutation` flag, if a `connector` was passed into the initializer for `Query`, then I don't think there'd be a need for the `Mutation` class. Also, testing would be simpler since there would be no need to stub out `Connector.new`.